data/team.yml

```diff
@@ -2202,7 +2203,7 @@ amy_johnson:
   description: Amy lives in Southern California with her husband and mini australian shepherd. When she isn't knee deep in spreadsheets, she enjoys hiking, camping, and snowboarding.
 
 director_design:
-  name: Christina Forney
+  name: Christina Forney, Jason Yavorska (Code Graph only)
```

Comment:
I understand Christina to be the Director of Design as per: https://docs.google.com/document/d/1s5cqXzN_oG26ye3lCEEAYbjtAQ6hi4JGzu6VyQe44yQ/edit#

Reply:
Ah, so you won't be reporting to me temporarily after all?

Reply:
I understand that you are my support for the transition to design team manager, but that I report to Christina.

Reply:
```suggestion
  name: Christina Forney
```